lib/ui/widgets/settingsView/about_widget.dart

```diff
@@ -84,7 +84,9 @@ class _AboutWidgetState extends State<AboutWidget> {
                         ),
                       ),
                       Text(
-                        'Arch: ${snapshot.data!['arch']}',
+                        snapshot.data!['supportedArch'].length > 1
+                            ? 'Supported Archs: ${snapshot.data!['supportedArch'].join(", ")}'
+                            : 'Supported Arch: ${snapshot.data!['supportedArch'].join(", ")}',
```

Comment:
What's the .join() for? you already check if it's a single architecture

Reply:
formatting, it's there for displaying `Supported Archs: arm-v8a, armabi-v7a` instead of `Supported Archs: [arm-v8a, armabi-v7a]`.

Reply:
formatting, it's there for displaying `Supported Archs: arm-v8a, armabi-v7a` instead of `Supported Archs: [arm-v8a, armabi-v7a]`.

Reply:
Do you need `.join()` for non-plural? because it would be `Supported arch: arm-v8a` for example, the `.join()` will never be required because its never above 1

Reply:
I realised that. It shouldn't affect anything but would be nice to fix. f94879c17195c8c0a7f8a2c291030f9be2078ca9